src/snowflake/cli/_plugins/nativeapp/run_processor.py

```diff
@@ -72,6 +72,22 @@
 }
 
 
+def print_warnings(create_or_upgrade_cursor: Optional[SnowflakeCursor]):
+    """
+    Shows warnings in the console returned by the CREATE or UPGRADE
+    APPLICATION command.
+    """
+    if not create_or_upgrade_cursor:
+        return
+
+    messages = [row[0] for row in create_or_upgrade_cursor.fetchall()]
+    warnings = [m for m in messages if m.lower().startswith("warning:")]
```

Comment:
Is it guaranteed that all warnings will start with this string?
Maybe you can add a sanity integration test?

Reply:
I don't think this requires an integration test; I don't want it to start failing if slightly different warnings / more warnings are added.

Reply:
Checking in on the `Warning:` thing now, maybe I can just show the "created successfully" message too (was trying to cut down on the amount of lines printed)